dev/ble_temperature.py

```diff
@@ -0,0 +1,77 @@
+# This example demonstrates a simple temperature sensor peripheral.
```

Comment:
We should probably go through these and remove comments talking about temperature as it's not really applicable to the project

Reply:
In fact, do we even need this file at all? Or is it just an example?

Reply:
Thanks for catching; the file is not used